modules/core/src/main/java/org/apache/ignite/configuration/ClientConfiguration.java

```diff
@@ -227,19 +230,34 @@ public ClientConfiguration setTcpNoDelay(boolean tcpNoDelay) {
     }
 
     /**
-     * @return Send/receive timeout in milliseconds.
+     * @return Connection timeout in milliseconds. 0 means infinite.
      */
-    public int getTimeout() {
```

Comment:
This is a breaking change, we don't remove public API parts like that. Please keep the old property, mark with `@Deprecated` annotation, make sure it still works.